internal/attestation/snp/validator.go

```diff
@@ -156,7 +156,14 @@ func addCRLtoVerifyOptions(attestationData *sevsnp.Attestation, verifyOpts *veri
 		return fmt.Errorf("could not parse CRL from attestation data: %w", err)
 	}
 
-	productLine := kds.ProductLine(attestationData.GetProduct())
+	// Attestation with v3 report: FMS is set in the report; Product is nil
+	// Attestation with v2 report: FMS is 0; Product is set
```

Comment:
See: https://github.com/edgelesssys/contrast/pull/1377#issuecomment-2801358817

Reply:
I observed this. `getAttestationWithoutCertChain` is not called, but `getAttestationFromKdsOrExtendedReport`, which leads to https://github.com/edgelesssys/go-sev-guest/blob/deaf0dfccf3c9643088729e6d51af10000b34ba3/verify/verify.go#L901-L903